src/media-gallery/media-gallery.js

```diff
@@ -123,16 +143,34 @@ component.template = `
             <span class="media-gallery__screen-reader-only">Remove</span>
         </button>
     </div>
+    <div class="media-list__media-container" ng-if="vm.view === vm.galleryView.LIST"
+         ng-repeat="media in vm.media" track by media.fileName>
+        <div class="col-md-12 media-list__row">
+            <div class="col-md-1" ng-click=""vm.viewLarger(media)">
```

Comment:
unless the column classes are provided by the theme library, we should assume they don't exist where this is being consumed. would that be ok for this element or should we add an additional class here?

Reply:
I was wondering the same, the example provides some styling but the new functionality needs a 
good amount more styling to be functional. I enhanced the comments with more styling to make this demo-able but its still lacking... What is your preferred approach? Get all of these classes into our themes, use lots of custom styling in the demo docs or leave the html more bare bones?

Reply:
@digitil Thoughts?